projects/services-ogc/src/lib/wps/test/wpsclient.spec.ts

```diff
@@ -247,18 +116,38 @@ describe(`Testing polling funcitonality`, () => {
     });
 
     const post1 = httpMockServer.expectOne(url1 + '?service=WPS&request=Execute&version=1.0.0&identifier=' + pId1);
-    post1.flush(createRequestAcceptedResponse(url1, pId1));
+    post1.flush(fakeWpsServer.createRequestAcceptedResponse(url1, pId1));
     const get1 = httpMockServer.expectOne(url1 + '?retrieveState');
-    get1.flush(createSuccessResponse(url1, pId1, 'outId1'));
+    get1.flush(fakeWpsServer.createSuccessResponse(url1, pId1, 'outId1'));
 
     const post2 = httpMockServer.expectOne(url2 + '?service=WPS&request=Execute&version=1.0.0&identifier=' + pId2);
-    post2.flush(createRequestAcceptedResponse(url2, pId2));
+    post2.flush(fakeWpsServer.createRequestAcceptedResponse(url2, pId2));
     const get2 = httpMockServer.expectOne(url2 + '?retrieveState');
-    get2.flush(createSuccessResponse(url2, pId2, 'outId2'));
+    get2.flush(fakeWpsServer.createSuccessResponse(url2, pId2, 'outId2'));
 
     httpMockServer.verify();
 
   }, 10000);
+
+  fit('#describeProcess should work as expected', (done) => {
```

Comment:
Shouldn't this method be **it** instead of **fit**

Reply:
Has been removed!